configs/nessie-0.6-iceberg-flink-0.11.yml

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (C) 2020 Dremio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# Nessie-Demo config-yaml to run demos against Nessie 0.5.1 and Iceberg 0.11.1 using Flink
+
+versions:
+  nessie: 0.6.0
+  iceberg: 652c175
+
+python_dependencies:
+  - pynessie==0.6.0
+  - jupyterlab==3.0.14
+  - apache-flink==1.12.1
+
+hadoop:
+  tarball: https://downloads.apache.org/hadoop/common/hadoop-2.10.1/hadoop-2.10.1.tar.gz
```

Comment:
Is that somehow contained in the `apache-flink` python library? If yes, we should use the stuff from there.
So the same mechanism as in `NessieDemoSpark.__init__`. If that works, would be great to get that functionality unified into a function in `_Util`.

Reply:
Unfortunately it's not included in `apache-flink` and users have to download/provide their own version of Hadoop

Reply:
Sad, okay